content/concepts/introduction.md

```diff
@@ -3,20 +3,23 @@ title: Introduction
 description: Ocean Protocol - Tools for the Web3 Data Economy
 ---
 
-In Ocean Protocol, each data service gets its own ERC20 **datatoken**. This enables data wallets, data exchanges, and data co-ops by directly leveraging crypto wallets, exchanges, and more.
+In Ocean Protocol, each data service gets its own ERC721 **data NFT** and one or more ERC20 **datatokens**. This enables data wallets, data exchanges, and data co-ops by directly leveraging crypto wallets, exchanges, and more.
```

Comment:
this is wrong, every asset gets its own Data NFT, not each service. Each service _can_ have one or more Datatokens, but our market does not allow this so we should not say that it is always like that.

Reply:
how about this: https://github.com/oceanprotocol/docs/pull/935